Update demo regarding fuel

v0d1ch opened the pull request:
## Why
We want to be able to run the demo successfully with the latest changes

## What

Try to run the demo and fix all the wrinkles

BONUS:
- We found a bug in the tui related to the sending amount on new transaction and fixed it.

---

<!-- Consider each and tick it off one way or the other -->
* [x] CHANGELOG updated or not needed
* [x] Documentation updated or not needed
* [x] Haddocks updated or not needed
* [x] No new TODOs introduced or explained herafter

v0d1ch committed:
Make demo work without marked fuel and update instructions

ch1bo commented:
@v0d1ch I was wrong, we _do_ use the `.vk` files in the `chainConfigFor` function. I reverted that part.

ch1bo commented on demo/seed-devnet.sh:
```diff
@@ -97,12 +88,14 @@ function publishReferenceScripts() {
+echo >&2 "Distributing funds to alice, bob and carol..."
+seedFaucet "alice-funds" 100000000 # 100 Ada to commit
+seedFaucet "bob-funds" 50000000 # 50 Ada to commit
+seedFaucet "carol-funds" 25000000 # 25 Ada to commit
```
@v0d1ch Updated this as you seem to have confused the comments. Also, I made every node have constant 20 ADA fuel (which should be enough for a single head life cycle)

ch1bo reviewed (approved):
Thanks for fixing my refactoring error on the TUI.

ch1bo committed:
Use unstable image in demo (for now)

We will bump this to the released tag on the next release according to
our workflow.
